xc7/tests/ddr/README.md

```diff
@@ -0,0 +1,16 @@
+# UART-DDR test
+
+This test is to verify the correctness of DDR modules.
+It comprehends:
+
+- UART-DDR design, which is the minimum design possible to test the correct functioning of the DDR module;
+- Testing script: under the `scripts` directory, there is a script that performs memory calibration, and is used to test the correctness of the design.
+
+## Testing
+
+To test the design, do the following:
+
+```
+cd scripts
+make test_dram
```

Comment:
I assume these instructions are from the prjxray version?  They don't apply here?

Reply:
Actually they do, but the Makefile is there to download the litex dependency needed by the test_sdram script.

I think that we could just avoid using the Makefile and call the script directly, and add the information in the README to get the litex dependency to let the test_sdram script to correctly run.